frontend/components/media/MediaVideo.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <!-- Note: mobile view -->
+  <div
+    class="md:hidden w-full h-full card-style bg-light-placeholder dark:bg-dark-placeholder flex justify-center items-center"
+  >
+    <div
+      class="p-6 bg-light-special-text dark:bg-dark-menu-selection rounded-full"
+    >
+      <IconCameraVideo
```

Comment:
Only real comment I have on this @otaree is that I'm thinking we don't need a custom icon component for this. We can use a normal `<Icon name='bi:camera-video' />` component or another one from [Bootstrap Icons](https://icones.js.org/collection/bi?s=video). We normally are making the custom icon components only in cases where we need to given that it's a branded icon or one we made specifically. We might event want to convert some of those custom icons over to ones available from [Icones](https://icones.js.org/), which is something you'd be welcome to work on, @otaree!

Reply:
I can definitely see how you'd think that one would be necessary though, so no stress! Looking at it, `IconJoin`, `IconLegal`, and maybe a few others could be converted. Let me know if you'd have interest in this!

Reply:
Hey @andrewtavis , Thank you for your thoughtful insights and suggestions. I'm definitely interested in working on the conversion of the custom icons to standard ones, including "IconJoin" and "IconLegal." To keep things organized and clear, would you prefer that I create a separate pull request for this icon conversion task? This way, it won't mix with the changes in the current pull request, and we can focus on each aspect more effectively. Please let me know your thoughts on this approach, and I'll be more than happy to get started on making these improvements.

Reply:
Yes, @otaree, that'd be perfect if it was in a separate PR. Also, if you'd be so kind as to make an issue for this this would also be great 😊 I'd comment in there with any other information for it that's needed like how we'd need to update the legal docs if we use a new icon pack, but it'd be good to have an issue for an overview.

Appreciate your willingness to help!